### ISSUE TITLE
agn_log_lbol has no effect on photometry across its whole prior under agn_model='skirtor'

### ISSUE THREAD

**Issue by suchethac** [suchethac/tengri#2069]
With `agn_model="skirtor"` and `agn_log_lbol` declared free, moving it across
its entire declared prior — `Uniform(9.0, 12.0)`, three decades of bolometric
luminosity — leaves the predicted photometry **bit-identical**.

## Measured

Spec is `tests/components/agn/test_agn_fused.py::TestSKIRTORPreintegration`'s
`skirtor_spec` (fixed z=0.3, SKIRTOR torus, everything else Fixed), filters are
its `simple_filters` (three top-hats over 3500-9000 A):

```
agn_log_lbol = 9.0   ->  [5.62066834e-26, 9.50522785e-26, 1.00075858e-25]
agn_log_lbol = 12.0  ->  [5.62066834e-26, 9.50522785e-26, 1.00075858e-25]
```

Equal to the last digit, so `jnp.all(phot_high > phot_low)` is False.

## Two candidate explanations, both worth confirming

1. **Expected under `cigale_joint`.** `agn_model="skirtor"` is deprecated and
   routes through composable blocks with `agn_norm="cigale_joint"` (the
   DeprecationWarning says so). Per CLAUDE.md that policy ties disc, torus and
   polar to CIGALE's single `agn_power` reference, so `agn_log_lbol` may
   legitimately carry no normalization on this path.

2. **The filters miss the component.** SKIRTOR is an IR torus (1-1000 um) and
   these filters are optical. The AGN *disc* does emit here, but under
   `cigale_joint` it is on `agn_power` too.

If (1) is the design, then a spec that declares `agn_log_lbol` free under
`cigale_joint` has a parameter with exactly zero effect on the likelihood — a
flat direction that every sampler will wander and no diagnostic will flag as
wrong. Either it should be refused at build time with the translation to
`agn_norm="independent"`, or the normalization should reach it.

Setting `agn_norm="independent"` on the same spec gives `[0., 0., 0.]` in these
bands, which does not distinguish the two either.

## Test status

`test_lbol_sensitivity` is marked `xfail(strict=True)` referencing this issue.
The test had never run: `_DATA_DIR` resolved to `tests/data`, so the file
skipped on every machine including CI.

**Comment by suchethac** [comment]
Claiming. Measured on main: with the spec as filed and `agn_ir_frac=0.122` the photometry is bit-identical across `agn_log_lbol` 9..12 (ratio 1.000 per band); with `agn_ir_frac=0.0` it moves (1.04-1.12). CLAUDE.md documents `cigale_joint` as tying disc/torus/polar to `agn_power`, active for torus=skirtor with fracAGN>0, so under that policy `agn_log_lbol` is inert for the disc and torus by design. It is NOT inert in general: the polar, NLR, BLR and FeII blocks read it directly (`runner.py` ~582-598, ~650-680); `recipes.agn_panchromatic()` rebuilt with SKIRTOR (polar and NLR free) moves every band by 0.33-1.2x across two decades. The flat direction exists exactly when all four consumers are inactive, which is the filed spec (`agn_polar_ebv=Fixed(0.0)`, no line blocks). Fix in flight: a build-time `ConfigError` for that case naming the three translations (fix `agn_log_lbol`, fix `agn_ir_frac=0.0`, or `agn_norm='independent'`); `test_lbol_sensitivity` runs un-xfailed with `agn_ir_frac` pinned at 0. The strict xfail only reproduced because `spec.sample()` draws `agn_ir_frac` at random -- the `agn_model="skirtor"` alias frees 74 AGN parameters for a spec that declared one, which is the #1482 class and out of scope here.
